build-tools/rel-build.sh

```diff
@@ -12,7 +12,7 @@ export BUILDDIR=$BUILDDIR
 
 go_install $(all_pkgs)
 
-ginkgo_test_with_coverage
+# ginkgo_test_with_coverage
```

Comment:
This shouldn't go in

Reply:
Correct. I'll double check the files to make sure only the necessary ones are part of the review.